docs/18_security_audit_document.md

```diff
@@ -210,6 +253,18 @@ Each rule is tested imitating any possible type of user. Consider the following
 
 The tests also cover invalid data input cases if the rule requires additional data validation.
 
+### Firebase Cloud Functions
+
+The [Firebase Cloud Functions](https://firebase.google.com/docs/firestore/extend-with-functions) component is responsible the [`build_days` collection](#the-build_days-collection) updates when the [`build` collection](#the-build-collection) changes. The `Firebase Cloud Functions` do not store any data, and just process the [`build` collection](#the-build-collection) updates.
+
+The `Firebase Cloud Functions` includes the two main functions: `onBuildAdded` and `onBuildUpdated`. Let's review them in a bit more details:
```

Comment:
Should we also describe the `validateEmailDomain` function here? Also, would it be great to have a bulleted list with all main `Cloud Functions` listed?